src/cclib/parser/psiparser.py

```diff
@@ -1009,6 +1009,59 @@ def get_normalization_factor(exp, lx, ly, lz):
             else:
                 assert self.moments[1] == dipole
 
+        ## Analytic Gradient (Psi4)
+         
+        #        -Total Gradient:
+        #   Atom            X                  Y                   Z
+        #  ------   -----------------  -----------------  -----------------
+        #     1       -0.000000000000     0.000000000000    -0.064527252292
+        #     2        0.000000000000    -0.028380539652     0.032263626146
+        #     3       -0.000000000000     0.028380539652     0.032263626146
+
+        if line.strip() == '-Total Gradient:':
+            next(inputfile)
+            next(inputfile)
+            line = next(inputfile).strip()
+            grads = []
+            while len(line) > 1:
+                idx, x, y, z = line.split()
+                grads.append((float(x), float(y), float(z)))
+                line = next(inputfile).strip()
+        
+            if not hasattr(self, 'grads'):
+                self.grads = []
+            self.grads.append(grads)
+
+        ## Finite Differences Gradient (Psi4)
+
+        # -------------------------------------------------------------
+        #   ## F-D gradient (Symmetry 0) ##
+        #   Irrep: 1 Size: 3 x 3
+        # 
+        #                  1                   2                   3
+        # 
+        #     1     0.00000000000000     0.00000000000000    -0.02921303282515
+        #     2     0.00000000000000    -0.00979709321487     0.01460651641258
+        #     3     0.00000000000000     0.00979709321487     0.01460651641258
+
+        if line.strip() == '## F-D gradient (Symmetry 0) ##':

```

Comment:
Do you know if this line contains anything other than `Symmetry 0`?

Reply:
As far as I know, the "Symmetry 0" specification is just because the gradient is saved and printed as a Psi4 Matrix object, which has a symmetry attribute. For gradients, it will always be Symmetry 0, but the symmetry specification is there for data which may contain symmetry blocks, e.g. molecular orbitals.

Reply:
Ok, just making sure. I didn't know if there were irrep-decomposed contributions to the gradient ever printed for some reason.